crypto-core/src/main/java/com/palantir/crypto2/io/ApacheCtrDecryptingSeekableInput.java

```diff
@@ -54,7 +54,7 @@ public static ApacheCtrDecryptingSeekableInput create(SeekableInput input, KeyMa
     /**
      * Returns true if the OpenSSL is able to be loaded.
      */
-    public boolean isSupported() {
+    public static boolean isSupported() {
```

Comment:
this name seems misleading. This class always works correctly, but only sometimes uses the native code, correct?